.github/workflows/sdist.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9"]
+        python-version: ["3.8", "3.9", "3.10-dev"]
```

Comment:
Python 3.10 is released yesterday. I wonder if GitHub Actions would have it soon.